src/app/main/connection-window/connection-window.service.ts

```diff
@@ -45,6 +45,14 @@ export class ConnectionWindowService {
     return configs;
   }
 
+  saveRememberPreference(rememberMe: boolean): void {
+    this.localStorageService.setItem('rememberMe', String(rememberMe));
+  }
+
+  isRemember(): boolean {
```

Comment:
`isRemember` does not make sense, maybe a better name can be `isRememberMeEnabled`

Reply:
done